Source/WebCore/editing/DeleteSelectionCommand.cpp

```diff
@@ -423,6 +429,9 @@ void DeleteSelectionCommand::saveTypingStyleState()
     RefPtr<Node> startNode = m_selectionToDelete.start().deprecatedNode();
     if (!startNode->isTextNode() && !startNode->hasTagName(imgTag) && !startNode->hasTagName(brTag))
         return;
+    
+    if (shouldNotInheritStyleFrom(*m_selectionToDelete.start().anchorNode()))
```

Comment:
Hm... in this case, we still want to keep the style outside of that container.
But that logic should probably go below where we're removing styles.

Reply:
By the way, this code should probably be using `deprecatedNode()` instead of `anchorNode()`:
https://rniwa.com/2011-06-27/position-and-anchor-types/

Reply:
I just pushed new build based on your suggestion - appreciate if you can have a look and see I did right changes. Thanks in advance!

Reply:
Yeah, something like that.

Reply:
Compiling now and running through EWS. Fingers crossed!